WIP- Aiven-for-Dragonfly-service

harshini-rangaswamy opened the pull request:
# What changed, and why it matters


The initial set of docs for Aiven for Dragonfly as a new service offering from Aiven.
 - docs/products/dragonfly.rst
 - docs/products/dragonfly/concepts/ha-dragonfly.rst
 - docs/products/dragonfly/get-started.rst
 
 
 Also now includes content from: 

-  https://github.com/aiven/devportal/pull/2334

- https://github.com/aiven/devportal/pull/2314

hoang-vo commented on docs/products/dragonfly/get-started.rst:
```diff
@@ -0,0 +1,56 @@
+Get started with Aiven for Dragonfly®
+=======================================
+
+The first step in using Aiven for Redis®* is to create a service. You can do so either using the `Aiven Web Console <https://console.aiven.io/>`_ or the `Aiven CLI <https://github.com/aiven/aiven-client>`_.
```
should be DF

hoang-vo commented on docs/products/dragonfly/get-started.rst:
```diff
@@ -0,0 +1,56 @@
+----------------------------------------------------
+1. Log in to the `Aiven Console <https://console.aiven.io/>`_.
+
+2. Follow :doc:`these instructions </docs/platform/howto/create_new_service>` to create a new Redis service.
```
same here

hoang-vo commented on docs/products/dragonfly/get-started.rst:
```diff
@@ -0,0 +1,56 @@
+
+`Aiven CLI <https://github.com/aiven/aiven-client>`_ provides a simple and efficient way to create an Aiven for Redis®* service. If you prefer launching a new service from the CLI, follow these steps:
+
+1. Determine the service plan, cloud provider, and region you want to use for your Redis®* service.
```
here

hoang-vo commented on docs/products/dragonfly/get-started.rst:
```diff
@@ -0,0 +1,56 @@
+`Aiven CLI <https://github.com/aiven/aiven-client>`_ provides a simple and efficient way to create an Aiven for Redis®* service. If you prefer launching a new service from the CLI, follow these steps:
+
+1. Determine the service plan, cloud provider, and region you want to use for your Redis®* service.
+2. Run the following command to create a Redis®* service named demo-Redis:
```
here

hoang-vo commented on docs/products/dragonfly/get-started.rst:
```diff
@@ -0,0 +1,56 @@
+
+Explore other resources for Aiven for Dragonfly
+--------------------------------------------------
+
```
this list below

hoang-vo commented on docs/products/dragonfly/get-started.rst:
```diff
@@ -0,0 +1,56 @@
+
+Learn how to connect to Aiven for Dragonfly using different programming languages:
+
+* redis-cli
```
redis-cli is compatible w DF, shall we make a point about it here so customer would not get confused?

harshini-rangaswamy commented on docs/products/dragonfly/concepts/test-concept.rst:
```diff
@@ -0,0 +1,5 @@
+Aiven for Dragonfly
+====================
+
+Content to be added!
```
This will be deleted! it was until more content was added.

ArthurFlageul commented on .github/vale/dicts/aiven.dic:
```diff
@@ -228,6 +230,8 @@ Savepoints
 savepoints
 schemaless
 scriptable
+SDK
+SDKs
```
You can also do this, regular expression style 👌 
```suggestion
SDKs?
```

safa-topal commented on docs/products/dragonfly/howto/migrate-aiven-redis-df-console.rst:
```diff
@@ -0,0 +1,112 @@
+
+   To prevent conflicts during replication:
+
+   * Avoid writing to tables in the target database that is currently undergoing migration.
```
User already can't write to target Dragonfly service as it will become read-only when migration is started. This is true for Redis migration as well, so I propose removing this line from both this PR and existing Redis doc. What do you think?

harshini-rangaswamy replied:
Update the text.

safa-topal commented on docs/products/dragonfly/howto/migrate-ext-redis-df-console.rst:
```diff
@@ -0,0 +1,116 @@
+  * **Password:** The password associated with the username.
+
+* Ensure firewalls allow traffic between databases or disable them temporarily.
+* The source Redis database must be SSL-secured, a standard migration requirement.
```
This is not true, SSL might be disabled on source service and migration will still work. How about changing this statement to a recommendation? It's best practice to use secure connection everywhere anyway.

harshini-rangaswamy replied:
Update text

safa-topal commented on docs/products/dragonfly/howto/migrate-ext-redis-df-console.rst:
```diff
@@ -0,0 +1,116 @@
+During the migration, you can:
+
+* Close the migration wizard by clicking **Close window** and later return to monitor the migration status from the service overview page.
+* The migration duration varies based on the size of your database. You can continue writing to the target database during migration but avoid manual changes that might impact replication.
```
This also is not true for both Redis and Dragonfly migrations. Target service becomes read-only so no data can be written there during migration.

harshini-rangaswamy replied:
Update the text.

safa-topal commented on docs/products/dragonfly/howto/migrate-ext-redis-df-console.rst:
```diff
@@ -0,0 +1,116 @@
+
+.. warning::
+   
+   * Stopping this migration will immediately halt the ongoing replication process, preserving the data already transferred to Aiven. You have the option to  initiate a new database migration at any time in the future, which will overwrite the entire database and its contents on Aiven with the latest data from the source.
```
extra space after `You have the option to`

harshini-rangaswamy replied:
Fixed

safa-topal reviewed (changes requested):
Lgtm overall, noticed some incorrect details seemingly present for already published Redis docs as well (assuming this is mostly copy paste from that doc)

harshini-rangaswamy commented:
> Lgtm overall, noticed some incorrect details seemingly present for already published Redis docs as well (assuming this is mostly copy paste from that doc)

Thanks, @safa-topal, I have addressed the feedback.

safa-topal reviewed (approved):
Lgtm
